docs/tables/alicloud_vpc_nat_gateway.md

```diff
@@ -0,0 +1,96 @@
+# Table: alicloud_vpc_nat_gateway
+
+NAT gateways are enterprise-class gateways that provide the Source Network Address Translation (SNAT) and Destination Network Address Translation (DNAT) features. Each NAT gateway provides a throughput capacity of up to 10 Gbit/s. NAT gateways also support cross-zone disaster recovery.
+
+## Examples
```

Comment:
Are there any examples we can add around checking NAT gateways that will expire in the next 30 days, or if they're only in 1 zone (this one would make more sense depending on what best practice is, e.g., NAT gateways should be in at least 2 zones)?